### ISSUE TITLE
Adding Parameter for HTTP Proxy and Port for an openvpn connection

### ISSUE THREAD

**Issue by mav-noob** [mullvad/mullvadvpn-app#6073]
### I have checked if others have suggested this already

- [X] I have checked this issue tracker to see if others have reported similar issues.

### Feature description

I'm working on a project where the only way for me to connect to
the internet is through a HTTP Proxy. I can't change this fact.
The OS is Ubuntu 22.04

As a VPN Service I want to use mullvad. But I can't authenticate my account or establish a vpn connection with the mullvad GUI.
Neither via the direct method nor via bridge mode.

Using the openvpn client and a customized mullvad configuration file (adding the http proxy parameter) works. 
So in general a openvpn connection can be established through the HTTP Proxy. Because of this it should be also possible to implement using a HTTP Proxy in the Mullvad GUI / Deamon.

In summary this feauture request focus on implementing and using a HTTP Proxy Adress and Port for an openvpn connection.

### Alternative solutions

As described in general a connection to the mullvad servers with the openvpn client through a HTTP Proxy ist possible.
To implement this feature in the GUI and the daemon would be ease the use of Mullvad for the user.

### Type of feature

- [ ] Better privacy/anonymity
- [X] Better at circumventing censorship
- [X] Easier to use
- [ ] Other

### Operating System

- [ ] Android
- [ ] iOS
- [ ] Windows
- [ ] macOS
- [X] Linux

**Comment by Rawa** [comment]
Closing as we are [sunsetting OpenVPN](https://mullvad.net/en/blog/removing-openvpn-15th-january-2026)
